gradient/api_sdk/clients/experiment_client.py

```diff
@@ -161,7 +155,7 @@ def create_multi_node(
 
         :param str name: Name of new experiment  [required]
         :param str project_id: Project ID  [required]
-        :param str experiment_type_id: Experiment Type ID [GRPC|MPI] [required]
+        :param int experiment_type_id: Experiment Type ID [GRPC|MPI] [required]
```

Comment:
GRPC/MPI ain't int

Reply:
Sure. But the experiment type id is. Fixing the comment